src/data/var/acceptData.js

```diff
@@ -7,7 +7,9 @@ export default function( owner ) {
 	//  - Node
 	//    - Node.ELEMENT_NODE
 	//    - Node.DOCUMENT_NODE
+	//    - Node.SHADOWROOT_NODE
```

Comment:
There's no such API as `Node.SHADOWROOT_NODE`. `nodeType === 11` represents `Node.DOCUMENT_FRAGMENT_NODE` and is shared by all document fragments.

Reply:
> nodeType === 11 represents Node.DOCUMENT_FRAGMENT_NODE and is shared by all document fragments.

Yes, So the condition `owner.host` was added to limit the [shadowroot](https://dom.spec.whatwg.org/#shadowroot)

Or Maybe `toString.call( obj ) === "[object ShadowRoot]"` better?

Reply:
My comment was mostly about the comment. :) These is not just a description, `Node.DOCUMENT_NODE` etc. are real APIs; you can type them in the browser console and you'll see they resolve to the underlying numbers. We just don't use these APIs but use numbers directly to save on size.

Meanwhile, there's no such thing as `Node.SHADOWROOT_NODE` - if you type it in the browser console, it will print `undefined` in all browsers.

See https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType

Reply:
got it :)

I just want to make it clear here that `nodeType === 11 && owner.node` limit the [shadowroot](https://dom.spec.whatwg.org/#shadowroot), and I'll change to `Node.DOCUMENT_FRAGMENT_NODE` first.

Reply:
> > nodeType === 11 represents Node.DOCUMENT_FRAGMENT_NODE and is shared by all document fragments.
> 
> 
> 
> Yes, So the condition `owner.host` was added to limit the [shadowroot](https://dom.spec.whatwg.org/#shadowroot)
> 
> 
> 
> Or Maybe `toString.call( obj ) === "[object ShadowRoot]"` better?